package.json

```diff
@@ -57,15 +60,19 @@
     "@storybook/addon-notes": "5.0.6",
     "@storybook/addon-options": "5.0.6",
     "@storybook/react": "5.0.6",
+    "@types/jest": "^24.0.11",
     "@types/react": "16.8.13",
     "babel-core": "6.26.3",
     "fork-ts-checker-webpack-plugin": "1.0.1",
     "gh-pages": "2.0.1",
+    "jest": "^24.7.1",
     "keyboardjs": "2.5.1",
     "markdown-loader": "5.0.0",
     "react": "16.8.4",
     "react-dom": "16.8.4",
+    "react-hooks-testing-library": "^0.4.0",
     "react-spring": "6.1.10",
+    "react-testing-library": "^6.1.2",
```

Comment:
Not a direct dependency anymore?

Reply:
Ah forgot to remove this from the first commit